modules/ROOT/pages/index.adoc

```diff
@@ -81,10 +81,6 @@ During development or after deployment, you can make changes to your integration
 * http://developer.mulesoft.com[MuleSoft developer site^]
 * https://training.mulesoft.com/[Training^]
 
-== Contributing to the Documentation
-
-To contribute to the English version of this documentation, select *Edit in Github* on the page you wish to modify, and submit a pull request to the git repo. Your improvements to MuleSoft documentation are appreciated by the entire community. Thank you!
-
 == Archived Documentation
```

Comment:
Does it make sense for this to follow "More Resources"?

Also, WHY does the audience care about doc archives? Will they need to access them? Should the title then be something more like Access Archive Documentation?

Oh, and one other question: this deliverable uses imperative titles for topics that aren't officially tasks, because they do not contain steps. Is this deliverable exempt from what we would require of most of our docs?

Reply:
This isn't hidden, it is deleted.